src/writeData/components/telegrafInputPluginsConfigurationText/nvidia_smi.conf

```
@@ -1,6 +1,8 @@
 # Pulls statistics from nvidia GPUs attached to the host
 [[inputs.nvidia_smi]]
-  ## Optional: path to nvidia-smi binary, defaults to $PATH via exec.LookPath
+  ## Optional: path to nvidia-smi binary, defaults "/usr/bin/nvidia-smi"
```

```suggestion
  ## Optional: path to nvidia-smi binary, defaults to "/usr/bin/nvidia-smi"
```

not sure if this is what you were going for or not...

---

It makes sense to add the "to" there but this change was pulled from the Telegraf repository. The change should be made there.